docs/RELEASE_NOTES.md

```diff
@@ -27,6 +27,18 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
 
+### 9.2.1 - Feb 02, 2023
```

Comment:
Do you want to change the date?

Reply:
It doesn't matter, I would leave it as it is. The changes for this release is done on 2.2.2023, so it's correct.

Reply:
But you will create a tag today. When user checks for release notes, he might gets confused.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/905/commits/8265aa7777489d12176f617e6c8b35b23920a156